tv_shows/styles.css

```
@@ -0,0 +1,158 @@
+body {
+    margin: 0;
+    width: 100%;
+    background-color: #2e0033;
+    color: white;
+
+}
+
+#page-container {
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    width: 60%;
```

Containere najčešće ovako nekako stiliziramo umjesto u postotcima
```suggestion
    width: 920px;
    max-width: 100%;
    padding: 12px;
```